### PR TITLE
[Java/el] Translated Java to Greek

### PR DESCRIPTION
**Pull request by lack3r**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

[Java/el] Translated Java to Greek

### CODE REVIEW

**Comment by adambard** [comment]
Ok, doesn't look like any greeks are coming by anytime soon to proofread. I'm sure it's probably fine. Thanks!

File: el-gr/java-gr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,858 @@
...
+language: java
+contributors:
+    - ["Jake Prather", "http://github.com/JakeHP"]
+    - ["Jakukyo Friel", "http://weakish.github.io"]
+    - ["Madison Dickson", "http://github.com/mix3d"]
+    - ["Simon Morgan", "http://sjm.io/"]
+    - ["Zachary Ferguson", "http://github.com/zfergus2"]
+    - ["Cameron Schermerhorn", "http://github.com/cschermerhorn"]
+    - ["Rachel Stiyer", "https://github.com/rstiyer"]
+filename: LearnJava.java
```
Comment (vendethiel) [inline el-gr/java-gr.html.markdown]: missing something here?

Comment (adambard) [inline el-gr/java-gr.html.markdown]: I changed it to LearnJava-gr after I merged

Comment (vendethiel) [inline el-gr/java-gr.html.markdown]: :+1:

**Pull request merged**